tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/adapter/variables/VariableVisibilityTest.java

```diff
@@ -194,8 +194,8 @@ public void globalVariableVisibilityTest() throws BallerinaTestException {
         // global constants
         debugTestRunner.assertVariable(globalVariables, "nameWithoutType", "\"Ballerina\"", "string");
         debugTestRunner.assertVariable(globalVariables, "nameWithType", "\"Ballerina\"", "string");
-        debugTestRunner.assertVariable(globalVariables, "nameMap", "(debug_test_resources/variable_tests" +
-                ":0:$anonType$nameMap$_0 & readonly)", "record");
+//        debugTestRunner.assertVariable(globalVariables, "nameMap", "(debug_test_resources/variable_tests" +
```

Comment:
Do we have any issues created to track this change? We need to uncomment it once it is fixed

Reply:
Yes we have  #40896, will put the issue in the test file also thanx.